setup.py

```diff
@@ -52,19 +52,19 @@ def run(self):
     "black",
     "numpy==1.21.0",
     "scipy",
-    "SimpleITK==2.1.0",
+    "SimpleITK!=2.0.*",
```

Comment:
Are we sure about this?

Reply:
Yup, confirmed with different checks. Our main dependency of SimpleITK is through TorchIO, and they have this dependency.

Reply:
Cool.